.codesandbox/tasks.json

```diff
@@ -0,0 +1,22 @@
+{
+	"setupTasks": [
+		"npm install",
+		"npm run build",
+		"node ./bin/prepare-codesandbox.js"
+	],
+	"tasks": {
+		"install-dependencies": {
+			"name": "Install Dependencies",
+			"command": "npm install"
+		},
+		"start-app": {
+			"name": "Run Dev Server",
```

Comment:
Maybe "Start WordPress server" would be better? I guess this could be confused with the webpack dev server (npm run dev)